app/controllers/api/v2/os_default_templates_controller.rb

```diff
@@ -63,12 +65,19 @@ def destroy
       private
 
       def rename_config_template
-        if !params[:config_template_id].nil?
+        if params[:config_template_id].present?
           params[:provisioning_template_id] = params.delete(:config_template_id)
           Foreman::Deprecation.api_deprecation_warning("Config templates were renamed to provisioning templates")
         end
       end
 
+      def rename_nested_config_template
+        if params[:os_default_template] && params[:os_default_template][:config_template_id].present?
```

Comment:
@domcleal Just adding  `wrap_parameters` didn't helped as it copies the param over. We have to rename both params the wrapped and the unwrapped anyway. If this is acceptable solution I'll fix the os controller the same way.